features/graphql/mutation.feature

```diff
@@ -323,29 +323,23 @@ Feature: GraphQL mutation support
     {
       "errors": [
         {
-          "message": "Field \"foo\" is not defined by type createDummyDtoNoInputInput.",
```

Comment:
@alanpoulain I think that this is better, see also the related change: https://github.com/api-platform/core/blob/e299566b5c4ee8c1c2791e50c5204146632435e9/src/GraphQl/Resolver/Factory/ItemMutationResolverFactory.php#L97-L99

WDYT?

Reply:
Then we should update the GraphQL Schema accordingly, and it will become impossible to trigger this code path.